src/test/kotlin/io/github/mojira/arisa/ModuleRegistryTest.kt

```diff
@@ -1,57 +1,64 @@
 package io.github.mojira.arisa
 
 import com.uchuhimo.konf.Config
+import com.uchuhimo.konf.source.yaml
 import io.github.mojira.arisa.infrastructure.config.Arisa
 import io.github.mojira.arisa.modules.Module
 import io.kotest.core.spec.style.StringSpec
+import io.kotest.matchers.collections.shouldContain
 import io.kotest.matchers.collections.shouldContainExactlyInAnyOrder
-import io.kotest.matchers.shouldBe
+import io.kotest.matchers.collections.shouldNotContain
 import org.reflections.Reflections
 
+val CONFIG = Config { addSpec(Arisa) }
+    .from.yaml.watchFile("config/config.yml")
+    .from.map.flat(
+        mapOf(
+            // Overwrite these settings to ensure we're always covering both test cases
+            "arisa.modules.attachment.enabled" to "true",
```

Comment:
Ideally we removed this, but well

Reply:
Yeah, I'm not quite happy either about having to use the real config file here and overwriting stuff but I couldn't quite figure out how to avoid this without specifying a completely separate test config file that would need to be updated along with the real one.

I guess this has the nice side effect of having a unit test that fails when the config isn't specified correctly?

Reply:
What exactly do you mean? I don't understand

Reply:
make sure that config.yml has arisa.modules.attachment.enabled as true and arisa.modules.missingCrash.enabled as false so we dont need to set it for the tests, we can disable in local.yml if we dont want it on production